filters/filters.go

```diff
@@ -58,6 +60,7 @@ func NewFilter(active... string) (*Filter, error) {
 			Services:          true,
 			Spaces:            true,
 			Stacks:            true,
+			Tasks:             true,
```

Comment:
Would you mind disabling the task collector by default ? Since this is a new collector and this collector could generate a very large number of metrics, I'd be more confortable if this feature is an opt-in option.

Maybe i'm overthinking, this is a suggestion and not a formal demand for change.

Reply:
Yup, good idea.